<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cadastro</class>
 <widget class="QWidget" name="Cadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>354</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stCadastro">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="3">
        <widget class="QPushButton" name="bt_Categ">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../recursos/pesquisa.png</normaloff>../recursos/pesquisa.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="bt_Filial">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../recursos/pesquisa.png</normaloff>../recursos/pesquisa.png</iconset>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="bt_Limpar">
         <property name="text">
          <string>Limpar</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../recursos/limpar.png</normaloff>../recursos/limpar.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="caixa_Filial"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="text_categ">
         <property name="text">
          <string>Categoria:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="text_Filial">
         <property name="text">
          <string>Filial:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="bt_Dep">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../recursos/pesquisa.png</normaloff>../recursos/pesquisa.png</iconset>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>504</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="bt_Secao">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../recursos/pesquisa.png</normaloff>../recursos/pesquisa.png</iconset>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="text_base">
         <property name="text">
          <string>Base:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="bt_Produto">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../recursos/pesquisa.png</normaloff>../recursos/pesquisa.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="caixa_Secao"/>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="caixa_Depart"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="text_Depart">
         <property name="text">
          <string>Departamento:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="caixa_Prod">
         <property name="text">
          <string>Produto:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="caixa_Categoria"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Cadastro Base/ Margem Cliente&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="caixa_Produto"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_Dep">
         <property name="text">
          <string>Seção:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QPushButton" name="bt_Salvar">
         <property name="text">
          <string>Salvar</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../recursos/salvar.png</normaloff>../recursos/salvar.png</iconset>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="caixa_Base"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bt_Limpar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Filial</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Limpar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Depart</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Limpar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Secao</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Limpar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Categoria</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Limpar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Produto</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Limpar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Base</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Salvar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Filial</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Salvar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Depart</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Salvar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Filial</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>691</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Salvar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Secao</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Salvar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Categoria</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Salvar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Produto</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Salvar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Base</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
